doc/fping.pod

```diff
@@ -38,6 +38,8 @@ Restrict name resolution and IPs to IPv6 addresses.
 
 Show systems that are alive.
 
+Not compatible with option B<--count> / B<-c>.
```

Comment:
How about adding something like "(Option B<-c> or B<-C> overrides B<-a>)"? This could start in the same line as the existing description.